docs/modules/ROOT/pages/guide/deploy-to-github-pages.adoc

```diff
@@ -0,0 +1,90 @@
+
+= Deploy documentation to GitHub Pages with Nix
```

Comment:
s/documentation/static website/? The latter is more general wording.

Reply:
Done, thanks!